api/python/tests/integration/test_packages.py

```diff
@@ -623,6 +623,8 @@ def test_s3_set_dir(self):
 
             list_object_versions_mock.assert_called_with('bucket', 'foo/')
 
+            pkg.set_dir('bar', 's3://bucket//foo')  # top-level '/' folder
```

Comment:
you should add a separate test case for that

Reply:
test_set_dir_root_folder_named_slash